### PR TITLE
feat(lsu): Change copy on LSU box.

### PR DESCRIPTION
**Pull request by keithamus**

### CODE REVIEW

File: views/partials/light-signup.html [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
-					<p class="o-email-only-signup__text">Get the FT’s essential daily email briefing of the best global news, comment & analysis</p>
+					<p class="o-email-only-signup__text">Get the FT’s essential daily email briefing of the best global news, comment & analysis delivered to your inbox every weekday for a week.</p>
 				</div>
 				<div class="o-email-only-signup__inner" id="o-email-only-signup-content" aria-hidden="false" data-o-email-only-signup-content>
 					<button class="o-email-only-signup__close" aria-controls="o-email-only-signup-content" data-o-email-only-signup-close data-trackable="close"><span class="o-email-only-signup__visually-hidden">Close</span></button>
 					<h3 class="o-email-only-signup__heading">Sign up to <span class="light-signup__brand"><em>First</em>FT</span></h3>
-					<p class="o-email-only-signup__text">Get the FT’s essential daily email briefing of the best global news, comment & analysis</p>
+					<p class="o-email-only-signup__text">Get the FT’s essential daily email briefing of the best global news, comment & analysis delivered to your inbox every weekday for a week.</p>
```
Comment (ironsidevsquincy) [inline views/partials/light-signup.html]: As your here, should probably `&amp` the &s too

**Comment by ironsidevsquincy** [comment]
👍

File: views/partials/light-signup.html (hunk 1 above)
Comment (keithamus) [inline views/partials/light-signup.html]: Did this, also `&apos`ed the `'`s

File: views/partials/light-signup.html [hunk 2]
```diff
@@ -4,12 +4,12 @@
 			<div class="light-signup--inner" data-o-grid-colspan="12 M11 L9">
 				<div class="o-email-only-signup__discreet o-email-only-signup__visually-hidden" aria-hidden="true" data-o-email-only-signup-discreet-content="">
 					<button class="o-email-only-signup__open" aria-controls="o-email-only-signup-content" data-o-email-only-signup-open data-trackable="open">Show more</button>
-					<p class="o-email-only-signup__text">Get the FT’s essential daily email briefing of the best global news, comment & analysis</p>
+					<p class="o-email-only-signup__text">Get the FT&apos;s essential daily email briefing of the best global news, comment &amp; analysis delivered to your inbox every weekday for a week.</p>
```
Comment (ironsidevsquincy) [inline views/partials/light-signup.html]: The apostrophes were those fancy `’` ones (a general next style decision), so no need to change them to entities

Comment (keithamus) [inline views/partials/light-signup.html]: Reverted.

**Pull request merged**